packages/evm/src/eof/stackDelta.ts

```diff
@@ -0,0 +1,472 @@
+  OpcodeInfo.unallocatedOpcode(0xbf),
+  OpcodeInfo.unallocatedOpcode(0xc0),
+  OpcodeInfo.unallocatedOpcode(0xc1),
+  OpcodeInfo.unallocatedOpcode(0xc2),
+  OpcodeInfo.unallocatedOpcode(0xc3),
+  OpcodeInfo.unallocatedOpcode(0xc4),
+  OpcodeInfo.unallocatedOpcode(0xc5),
+  OpcodeInfo.unallocatedOpcode(0xc6),
+  OpcodeInfo.unallocatedOpcode(0xc7),
+  OpcodeInfo.unallocatedOpcode(0xc8),
+  OpcodeInfo.unallocatedOpcode(0xc9),
+  OpcodeInfo.unallocatedOpcode(0xca),
+  OpcodeInfo.unallocatedOpcode(0xcb),
+  OpcodeInfo.unallocatedOpcode(0xcc),
+  OpcodeInfo.unallocatedOpcode(0xcd),
+  OpcodeInfo.unallocatedOpcode(0xce),
+  OpcodeInfo.unallocatedOpcode(0xcf),
+  OpcodeInfo.validOpcode('DATALOAD', 0xd0, 1, 1, 1),
+  OpcodeInfo.validOpcode('DATALOADN', 0xd1, 0, 1, 3),
+  OpcodeInfo.validOpcode('DATASIZE', 0xd2, 0, 1, 1),
+  OpcodeInfo.validOpcode('DATACOPY', 0xd3, 3, 0, 1),
+  OpcodeInfo.unallocatedOpcode(0xd4),
+  OpcodeInfo.unallocatedOpcode(0xd5),
+  OpcodeInfo.unallocatedOpcode(0xd6),
+  OpcodeInfo.unallocatedOpcode(0xd7),
+  OpcodeInfo.unallocatedOpcode(0xd8),
+  OpcodeInfo.unallocatedOpcode(0xd9),
+  OpcodeInfo.unallocatedOpcode(0xda),
+  OpcodeInfo.unallocatedOpcode(0xdb),
+  OpcodeInfo.unallocatedOpcode(0xdc),
+  OpcodeInfo.unallocatedOpcode(0xdd),
+  OpcodeInfo.unallocatedOpcode(0xde),
+  OpcodeInfo.unallocatedOpcode(0xdf),
+  OpcodeInfo.terminalOpcode('RJUMP', 0xe0, 0, 0, 3),
+  OpcodeInfo.validOpcode('RJUMPI', 0xe1, 1, 0, 3),
+  OpcodeInfo.validOpcode('RJUMPV', 0xe2, 1, 0, 2),
+  OpcodeInfo.validOpcode('CALLF', 0xe3, 0, 0, 3),
+  OpcodeInfo.terminalOpcode('RETF', 0xe4, 0, 0, 1),
+  OpcodeInfo.terminalOpcode('JUMPF', 0xe5, 0, 0, 3),
+  OpcodeInfo.validOpcode('DUPN', 0xe6, 0, 1, 2),
+  OpcodeInfo.validOpcode('SWAPN', 0xe7, 0, 0, 2),
+  OpcodeInfo.validOpcode('EXCHANGE', 0xe8, 0, 0, 2),
+  OpcodeInfo.unallocatedOpcode(0xe9),
+  OpcodeInfo.unallocatedOpcode(0xea),
+  OpcodeInfo.unallocatedOpcode(0xeb),
+  OpcodeInfo.validOpcode('EOFCREATE', 0xec, 4, 1, 2),
+  OpcodeInfo.unallocatedOpcode(0xed),
+  OpcodeInfo.terminalOpcode('RETURNCONTRACT', 0xee, 2, 1, 2),
+  OpcodeInfo.unallocatedOpcode(0xef),
+  OpcodeInfo.invalidOpcode('CREATE', 0xf0),
+  OpcodeInfo.invalidOpcode('CALL', 0xf1),
+  OpcodeInfo.invalidOpcode('CALLCODE', 0xf2),
+  OpcodeInfo.terminalOpcode('RETURN', 0xf3, 2, 0, 1),
+  OpcodeInfo.invalidOpcode('DELEGATECALL', 0xf4),
+  OpcodeInfo.invalidOpcode('CREATE2', 0xf5),
+  OpcodeInfo.unallocatedOpcode(0xf6),
+  OpcodeInfo.validOpcode('RETURNDATALOAD', 0xf7, 1, 1, 1),
+  OpcodeInfo.validOpcode('EXTCALL', 0xf8, 4, 1, 1),
+  OpcodeInfo.validOpcode('EXTDELEGATECALL', 0xf9, 3, 1, 1),
+  OpcodeInfo.invalidOpcode('STATICCALL', 0xfa),
+  OpcodeInfo.validOpcode('EXTSTATICCALL', 0xfb, 3, 1, 1),
+  OpcodeInfo.unallocatedOpcode(0xfc),
+  OpcodeInfo.terminalOpcode('REVERT', 0xfd, 2, 0, 1),
+  OpcodeInfo.terminalOpcode('INVALID', 0xfe, 0, 0, 1),
+  OpcodeInfo.invalidOpcode('SELFDESTRUCT', 0xff),
+]
+
+console.log(JSON.stringify(stackDelta))
+
+
+*/
```

Comment:
Maybe we put this generation code into `evm/scripts` and then reference from here? 🤔

Reply:
Yup, good point, will do!

Reply:
Still open task

Reply:
Have moved this to a script!